lib/src/main/kotlin/com/swmansion/starknet/data/types/Payloads.kt

```diff
@@ -83,6 +89,21 @@ internal data class GetNoncePayload(
     override val blockId: BlockId,
 ) : PayloadWithBlockId()
 
+@Serializable
+internal data class GetStorageProofPayload @JvmOverloads constructor(
+    @SerialName("block_id")
+    val blockId: BlockId,
```

Comment:
I don't think `BlockId` is needed:
https://github.com/starkware-libs/starknet-specs/blob/647caa00c0223e1daab1b2f3acc4e613ba2138aa/api/starknet_api_openrpc.json#L910

Reply:
It is, some time ago they added a PR updating storage proof (sorry, I didn't link it in the PR description)
https://github.com/starkware-libs/starknet-specs/pull/232

Reply:
Got it, was just checking out the changes from the tag, not branch 😅